guides/source/active_record_postgresql.md

```diff
@@ -290,7 +290,7 @@ SELECT n.nspname AS enum_schema,
 ### UUID
 
 * [type definition](https://www.postgresql.org/docs/current/static/datatype-uuid.html)
-* [pgcrypto generator function](https://www.postgresql.org/docs/current/static/pgcrypto.html#id-1.11.7.35.7)
+* [pgcrypto generator function](https://www.postgresql.org/docs/current/static/pgcrypto.html#id-1.11.7.34.7)
```

Comment:
`#id-1.11.7.34.7` is "PGP Encryption Functions", is it intentional?
Just remove the anchor if there is no proper anchor.

Reply:
Sorry, it's not intentional. I fixed it to remove the anchor.